### PR TITLE
Update 0.18.0 changelog

### PR DESCRIPTION
**Pull request by testforstephen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.18.0 - 2019-04-16
+### Added
+- Support "Logical Structure" for debug variables (e.g. Map and Collection classes). [#227](https://github.com/Microsoft/vscode-java-debug/issues/227).
```
Comment (andxu) [inline CHANGELOG.md]: debug => debugging?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
...
+- Jump to source code when clicking the stack trace info in Debug Console. [#490](https://github.com/Microsoft/vscode-java-debug/issues/490), [#506](https://github.com/Microsoft/vscode-java-debug/issues/506).
```
Comment (andxu) [inline CHANGELOG.md]: when => by

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
...
+- Automatically add `--enable-preview` to vmArgs when necessary. [#553](https://github.com/Microsoft/vscode-java-debug/issues/553).
+
+### Changed
+- Disable `java.debug.settings.showStaticVariables` by default to not show the static fields for debug variables.
```
Comment (andxu) [inline CHANGELOG.md]: remote `debug`

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
...
+
+### Fixed
+- Fix: Debug Console does not autocomplete when a .class file is open. [#535](https://github.com/Microsoft/vscode-java-debug/issues/535).
```
Comment (andxu) [inline CHANGELOG.md]: support autocomplete

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,15 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## 0.18.0 - 2019-04-16
 ### Added
-- Support "Logical Structure" for debug variables (e.g. Map and Collection classes). [#227](https://github.com/Microsoft/vscode-java-debug/issues/227).
-- Jump to source code when clicking the stack trace info in Debug Console. [#490](https://github.com/Microsoft/vscode-java-debug/issues/490), [#506](https://github.com/Microsoft/vscode-java-debug/issues/506).
+- Enable "Logical Structure" view for Map and Collection variables. [#227](https://github.com/Microsoft/vscode-java-debug/issues/227).
+- Add the stacktrace source link support in Debug Console output. [#490](https://github.com/Microsoft/vscode-java-debug/issues/490), [#506](https://github.com/Microsoft/vscode-java-debug/issues/506).
```
Comment (yaohaizh) [inline CHANGELOG.md]: Support the stack trace source link in the Debug Console output.

**Pull request merged**